cpu/cc26xx-cc13xx/rf-core/rf-rat-monitor.c

```diff
@@ -136,9 +138,9 @@ static void
 handle_rat_overflow(void *unused)
 {
   if(rf_core_is_accessible()) {
-  uint_fast8_t success = 0;
       //* if RF is powerdown, nothing to check
-  success = rf_rat_check_overflow(false);
+      WHEN_ANNOTATE(uint_fast8_t success = )

```

Comment:
Does the warning stop when you just add this: `(void) success;` ?
IMHO this macro is calling for trouble and is not exactly straight forward to read :)